documents/Libraries/sxpbrlib/sxpbrlib_defs.mtlx

```diff
@@ -137,6 +137,16 @@
     <input name="weight" type="float" value="1.0" doc="Mixing weight, range [0, 1]."/>
   </nodedef>
 
+  <!--
+    Node: <addbsdf>
+    A node for additive blending of BSDF's.
+  -->
+  <nodedef name="ND_addbsdf" node="addbsdf" type="BSDF" nodegroup="pbr"
+           doc="A node for additive blending of BSDF's.">
+    <input name="in1" type="BSDF" doc="First BSDF."/>
+    <input name="in2" type="BSDF" doc="Second BSDF."/>
+  </nodedef>
```

Comment:
Do a simple additive blend of the input nodes. Each BSDF lobe in ShaderX has a weight input already, controlling it's contribution. So we don't really need additional weights for each input here. Instead a used can control the weights directly on the connected nodes. (see the test example below)